google/cloud/forseti/services/server_config/service.py

```diff
@@ -26,6 +29,48 @@
 LOGGER = logger.get_logger(__name__)
 
 
+def _run(client):
+    """Runs Forseti.
+
+    Args:
+        client (ClientComposition): Forseti client object.
+
+    Returns:
+        ServerRunReply: the returned proto message.
+    """
+    today = datetime.now()
+    model_name = today.strftime('run_model_%Y%m%d_%H%M')
+    message = 'Forseti server run complete'
+
+    inventory_id = 0
+
+    for progress in client.inventory.create():
```

Comment:
Should you check for the status of the inventory, so that if there are errors then don't proceed to build a model or the rest of the e2e run?

Reply:
That's probably a good idea.

Reply:
Done